admin/users/views.py

```diff
@@ -0,0 +1,31 @@
+from django.shortcuts import redirect
+
+from website.project.model import User
+
+from admin.base.views import GuidFormView, GuidView
+from admin.users.templatetags.user_extras import reverse_user
+from .serializers import serialize_user
+
+
+def remove_2_factor(request, guid):
```

Comment:
See comment https://github.com/CenterForOpenScience/osf.io/pull/5041/files#r54485386 : this method may need refactoring.